Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -192,12 +192,12 @@ namespace AzToolsFramework::Prefab
         }
 
         // Close all container entities in the old path.
-        CloseInstanceContainers(m_instanceFocusHierarchy);
+        SetInstanceContainersOpenState(m_instanceFocusHierarchy, false);
 
-        AZ::EntityId previousContainerEntityId = m_focusedInstanceContainerEntityId;
+        RootAliasPath previousContainerRootAliasPath = m_focusedInstanceRootAliasPath;
```

Comment:
I'm not familiar with the APi calls in this section but could `previousContainerRootAliasPath` be `const`? Same for any others in this section that don't obviously mutate.